site/ja/xla/shapes.md

```diff
@@ -0,0 +1,113 @@
+# シェイプとレイアウト
+
+XLA `shape` proto（[xla_data.proto](https://www.tensorflow.org/code/tensorflow/compiler/xla/xla_data.proto)） は、N次元配列（略して配列）のランクとサイズ、データ型について記述します。
+
+
+## 用語、表記、決まりごと
+
+* 配列のランクは次元の数とおなじです。正しい配列のランクは、次元数が1以上の配列のことを指します。
```

Comment:
[suggestion] The original document emphasizes the word as `*true rank*`, so it might be better to follow the same way.

Reply:
Nice catch! I fixed it.